firmware_configurations/klipper/TechnoPhreak/BTT_SKR_v2/printer.cfg

```diff
@@ -0,0 +1,608 @@
+##	Increasing position_endstop brings nozzle closer to the bed
+##  After you run Z_ENDSTOP_CALIBRATE, position_endstop will be stored at the very end of your config
+position_endstop: -0.5
+##--------------------------------------------------------------------
+
+##	Uncomment below for 250mm build
+#position_max: 230
+
+##	Uncomment below for 300mm build
+#position_max: 280
+
+##	Uncomment below for 350mm build
+#position_max: 330
+
+##--------------------------------------------------------------------
+position_min: -5
+homing_speed: 8
+second_homing_speed: 3
+homing_retract_dist: 3
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z]
+uart_pin: z:PE0
+interpolate: true
+run_current: 0.8
+hold_current: 0.8
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+##	Z MCU - In Y Position
+##	Z1 Stepper - Rear Left
+[stepper_z1]
+step_pin: z:PD5
+dir_pin: z:PD4
+enable_pin: !z:PD6
+rotation_distance: 40
+gear_ratio: 80:16
+microsteps: 16
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z1]
+uart_pin: z:PD3
+interpolate: true
+run_current: 0.8
+hold_current: 0.8
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+##	Z MCU - In Z Position
+##	Z2 Stepper - Rear Right
+[stepper_z2]
+step_pin: z:PA15
+dir_pin: !z:PA8
+enable_pin: !z:PD1
+rotation_distance: 40
+gear_ratio: 80:16
+microsteps: 16
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z2]
+uart_pin: z:PD0
+interpolate: true
+run_current: 0.8
+hold_current: 0.8
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+##	Z MCU - In E0 Position
+##	Z3 Stepper - Front Right
+[stepper_z3]
+step_pin: z:PD15
+dir_pin: z:PD14
+enable_pin: !z:PC7
+rotation_distance: 40
+gear_ratio: 80:16
+microsteps: 16
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z3]
+uart_pin: z:PC6
+interpolate: true
+run_current: 0.8
+hold_current: 0.8
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+
+#####################################################################
+# 	Extruder
+#####################################################################
+
+#	E0 on MCU X/Y
+[extruder]
+step_pin: PD15
+dir_pin: !PD14
+enable_pin: !PC7
+##	Update value below when you perform extruder calibration
+##	If you ask for 100mm of filament, but in reality it is 98mm:
+##	rotation_distance = <previous_rotation_distance> * <actual_extrude_distance> / 100
+##  22.6789511 is a good starting point
+rotation_distance: 22.6789511	#Bondtech 5mm Drive Gears
+##	Update Gear Ratio depending on your Extruder Type
+##	Use 50:17 for Afterburner/Clockwork (BMG Gear Ratio)
+##	Use 80:20 for M4, M3.1
+gear_ratio: 50:17				#BMG Gear Ratio
+microsteps: 16
+full_steps_per_rotation: 200	#200 for 1.8 degree, 400 for 0.9 degree
+nozzle_diameter: 0.400
+filament_diameter: 1.75
+heater_pin: PB3
+##	Validate the following thermistor type to make sure it is correct
+sensor_type: ATC Semitec 104GT-2

```

Comment:
Please comment this line out to force the user to choose the correct thermistor. Also add this line `##  See https://www.klipper3d.org/Config_Reference.html#common-thermistors for additional options` above.

Reply:
Should be good now